_episodes/00-intro.md

```diff
@@ -5,6 +5,7 @@ questions:
 - "How is using a cluster different than your laptop?"
 keypoints: 
 - "A cluster or distributed system is useful when your computations require more than one computer." 
+- "Because a cluster is distributed, it is only useful for certain types of computational problems."
```

Comment:
not sure if this formulation catches it, but I am heavily biased on the other hand. how about:
```
- "Because a cluster is distributed, it is extremely useful for many types of computational problems, but not all."